server/src/main/java/com/testsigma/service/AgentExecutionService.java

```diff
@@ -1413,6 +1413,10 @@ public List<TestCaseStepEntityDTO> getExecutableTestSteps(WorkspaceType workspac
           }
           //TODO: check logic for test step key Generation and recursive logic for step group generation
```

Comment:
This TODO comment is not needed here. Remove this

Reply:
removed